homeassistant/components/cover/opengarage.py

```diff
@@ -38,6 +38,9 @@
     1: STATE_OPEN,
 }
 
+""" Update cycles before the internal move state will reset."""
+STATE_RESET_COUNTER = 2
```

Comment:
What does platform needs is a standalone library. We don't want this kind of device specific logic inside home assistant. Please extract it, including the requests to a standalone library published on pypi.

https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#6-communication-with-devices-services

Reply:
I'm not going to rewrite the component to a library for a small change. This is a bit ridiculous. If that is required please just close this PR and I'll notify users in the forum how to fix it on their side via custom_components folder.